tests/unit/Trace/TracerFactoryTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace OpenTelemetry\Tests\Trace;
+
+use Error;
+use OpenTelemetry\Context\SpanContext;
+use OpenTelemetry\Trace\TracerFactory;
+use PHPUnit\Framework\TestCase;
+
+class TracerFactoryTest extends TestCase
+{
+    /**
+     * @test
+     * @expectedException Error
+     */
+    public function shouldNotBeAbleToInstantiateDirectly()
+    {
+        new TracerFactory();
+    }
+}
```

Comment:
What other tests are you considering including for the TracerFactory?  A `getTracer` and `getInstance` test might be prudent?

Reply:
Yes, of course. This is work in progress.